redbot/core/events.py

```diff
@@ -97,10 +99,54 @@ async def on_ready():
                     "Outdated version! {} is available "
                     "but you're using {}".format(data["info"]["version"], red_version)
                 )
-                outdated_red_message = (
+                outdated_red_message = _(
                     "Your Red instance is out of date! {} is the current "
                     "version, however you are using {}!"
                 ).format(data["info"]["version"], red_version)
+                requires_python = data["info"]["requires_python"]
+                current_python = platform.python_version()
+                if expected_version(current_python, requires_python):
+                    extra_update = _(
+                        "\n\nTo update your bot, first shutdown your "
+                        "bot then open a window of {console} (Not as admin) and "
+                        "run the following:\n\n"
+                    ).format(console=platform_console)
+                    extra_update += '```"{python}" -m pip install -U Red-DiscordBot```'.format(
+                        python=sys.executable
+                    )
+                    extra_update += _(
+                        "\n\nPlease be aware that the command above is a best guess;"
+                        "\nThis is based on your current OS, enviroment and Python version and "
+                        "there is a small change this could be incorrect."
+                        "\n**Should the above not work for you please check our docs at <{docs}> "
```

Comment:
```suggestion
                        "\nIt is based on your current OS, environment, and Python version so "
                        "there is a small chance it could be incorrect."
                        "\n**Should the above not work for you, please check our docs at <{docs}> "
```